packages/nest/src/migrations/update-13-2-0/update-13-2-0.ts

```diff
@@ -0,0 +1,45 @@
+import { chain, Rule, SchematicContext } from '@angular-devkit/schematics';
```

Comment:
suggestion: let's switch to use `nrwl/devkit` instead. 
![image](https://user-images.githubusercontent.com/25516557/139494891-c89c2d1a-36de-4100-8d41-77d4ba4ad531.png)

If you have this box checked in the PR, I can push to switch to `nrwl/devkit`

Reply:
Is checked 😄 
<img width="298" alt="Screen Shot 2021-10-30 at 12 05 28 AM" src="https://user-images.githubusercontent.com/371939/139518176-aff55649-cde9-49fa-bbf8-18674fcc9dba.png">